modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

```diff
@@ -62,9 +39,51 @@ spec:
 For OpenShift versions before 4.15, the `io.openshift.podman-fuse: ""` annotation is also required.
 ====
 
+. If you are using a custom image for the workspace container, create the `/home/user/.config` folder and configure the `storage.conf` file on runtime via the entrypoint.
+To do this, create add the following to the workspace container image's entrypoint script before building the image.
++
+====
+[source,bash,subs="+quotes,+macros"]
```

Comment:
please don't use bash as a source language, it causes issues https://redhat-documentation.github.io/supplementary-style-guide/#code-example-syntax-highlighting

Reply:
Fixed